code/modules/mob/living/carbon/human/human.dm

```diff
@@ -745,10 +745,13 @@
 	dna.check_integrity(src)
 	return
 
-/mob/living/carbon/human/get_species()
+/mob/living/carbon/human/get_species(var/reference = 0)

```

Comment:
I extended the get species function in a nondestructive way
It's pretty dumb that it returns the name rather than a reference